kloppy/tests/test_statsbomb.py

```diff
@@ -917,6 +914,13 @@ def test_keeper_sweeper(self, dataset: EventDataset):
             GoalkeeperActionType.PICK_UP
         )
 
+    def test_under_pressure(self, dataset: EventDataset):
```

Comment:
I think it would be good if we added a few more tests. For instance, check the number of under-pressure events and check whether a non-under-pressure event doesn't have the qualifier.